### PR TITLE
Updates related to logging into converted mailbox

### PR DESCRIPTION
**Pull request by ninobb**
It was not explicitly clear that the user's old username / password actually WILL be able to log into the converted shared mailbox after conversion. Added information on resetting the password (if user was deleted) and what other things need to be done if this is an employee that has left the organization.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Here's how you convert a mailbox to a shared mailbox:
  ![covnerttoshared](https://user-images.githubusercontent.com/45987684/50208599-966a3a80-0397-11e9-8148-8d2595343709.PNG)
   
 
-5. If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. The user won't be able to sign in using the old account.
+5. If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. Note that if you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. Please see [Remove a former employee from Office 365](https://docs.microsoft.com/en-us/office365/admin/add-users/remove-former-employee).
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]: If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox, as the shared mailbox needs it as an anchor. Note that if you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. Please see [Remove a former employee from Office 365](https://docs.microsoft.com/en-us/office365/admin/add-users/remove-former-employee).

Comment (mypil) [reply Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]: @ninobb -  we need your kind assistance to merge the new commit for the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by dianef77** [comment]
@dianef77  please decide

**Comment by mypil** [comment]
@dianef77 - Do you have any updates on this?

Thank you.

**Comment by mypil** [comment]
@dianef77 - I would like to follow up on this. Can you please assist to check, approve and merge the content of this PR?

Thank you.

cc: @aruhdam @kenwith

**Comment by cmcatee-MSFT** [comment]
@ninobb Please delete "Note that" and begin the sentence with "If you are..." Notes are a specific type of information which is called out differently in the formatting. In this case, a call-out is not necessary, and the information can just be incorporated into the main paragraph. Thanks.

**Comment by mypil** [comment]
@ninobb - Please assist to apply the recommended changes by cmcatee-MSFT so the PR can be merged. Thanks.

**Comment by ninobb** [comment]
@mypil and @cmcatee-MSFT - I made the change you requested

**Pull request merged**